dsfhub/config.go

```diff
@@ -46,3 +55,12 @@ func (c *Config) Client() (interface{}, error) {
 
 	return client, nil
 }
+
+func IsValidSyncType (sync_type string) bool {
+	for _, valid_sync_type := range validSyncTypes {
+		if sync_type == valid_sync_type {
+			return true
+		}
+	}
+	return false
```

Comment:
We could also use [slices.Contains](https://pkg.go.dev/golang.org/x/exp/slices#Contains) here, have you given it a try?
`return slices.Contains(validSyncTypes, sync_type)` I think Uncle Bob would approve of the function name though! 🦾

Reply:
hadn't, but that's a hell of a lot cleaner.